timesketch/lib/tasks.py

```diff
@@ -484,3 +486,52 @@ def run_csv_jsonl(source_file_path, timeline_name, index_name, source_type):
     _set_timeline_status(index_name, status='ready')
 
     return index_name
+
+
+@celery.task(track_started=True, base=SqlAlchemyTask)
+def run_mans(source_file_path, timeline_name, index_name, source_type):
+    """Create a Celery task for processing mans file.
+
+    Args:
+        source_file_path: Path to mans file.
+        timeline_name: Name of the Timesketch timeline.
+        index_name: Name of the datastore index.
+        source_type: Type of file, csv or jsonl.
+
+    Returns:
+        Name (str) of the index.
+    """
+    # Log information to Celery
```

Comment:
this comment provides no value, just remove it

Reply:
let me know the method you want to use: keep both comments psort and mans to es, or remove both